MapboxNavigation/NavigationViewController.swift

```diff
@@ -537,7 +539,7 @@ extension NavigationViewController: RouteMapViewControllerDelegate {
     }
     
     func mapViewController(_ mapViewController: RouteMapViewController, roadNameAt location: CLLocation) -> String? {
-        return delegate?.navigationViewController?(self, roadNameAt: location)
+        return delegate?.navigationViewController?(self, roadNameAt: location) ?? ""
```

Comment:
This method never returns `nil`, whereas the return type is optional because we do need it to return `nil` in the case where `navigationViewController(_:roadNameAt:)` is unimplemented.

Reply:
Hey, @1ec5 thanks for catching. 🤦‍♂️  I missed this during the refactor to address the prior comments.